oktoberfest/preprocessing/preprocessing.py

```diff
@@ -267,6 +267,8 @@ def convert_search(
     input_path: Union[str, Path],
     search_engine: str,
     tmt_label: str = "",
+    stat_modifications: Dict[str, Tuple[str, float]] = None,
```

Comment:
the type annotation is wrong, this is an Optional[Dict[str, Tuple[str, float]]]

Reply:
you are also missing the documentation of var and stat modifications params in the docstring

Reply:
also please rename the variables into static_mods, var_mods